float64 scale_ratio
---
bool success 
string message
